packages/fxa-settings/src/components/TotpInputGroup/index.tsx

```diff
@@ -21,6 +21,9 @@ type SingleInputProps = {
   index: number;
 };
 
+const NUMBERS_ONLY_REGEX = /^[0-9]+$/;
+const numbersOnlyRegex = new RegExp(NUMBERS_ONLY_REGEX);
```

Comment:
Isn't `NUMBERS_ONLY_REGEX` already a regex? Why do we need to rewrap in a `new RegExp(...)` constructor?
(although possibly moot since I see that was old code down below from L117-118.)